alerta-covid/spec/requests/api/v1/areas_spec.rb

```diff
@@ -0,0 +1,16 @@
+ require 'rails_helper'
+
+RSpec.describe "/areas", type: :request do
+  
+  let!(:institution) { FactoryBot.create(:institution) }
+  let!(:area) { FactoryBot.create(:area, institution_id: institution.id) }
```

Comment:
You do not need to be that explicit when using `FactoryBot`, you can use `create` directly 😄 

Try it 👍🏽

Reply:
Thank you I will take this observation into account for new Issues. 😅